### PR TITLE
AT 11.01.01 | New view > Create new view > Verify that new view was created

### PR DESCRIPTION
**Pull request by BabintsevaS**
[https://trello.com/c/Eta9UNbC](url)

### CODE REVIEW

**Comment by BabintsevaS** [comment]
[AT 11.01.01 | New view > Create new view > Verify that new view was created](https://trello.com/c/Eta9UNbC/305-at-110101-new-view-create-new-view-verify-that-new-view-was-created)

**Review by SergeiKuznetsov — APPROVED** [review]

File: src/test/java/NewViewTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        getDriver().findElement(By.xpath("//*[@id='projectstatus-tabBar']/div/div[1]/div[2]/a")).click();
+        getDriver().findElement(By.cssSelector("svg.icon-edit-delete")).click();
+        getDriver().findElement(By.id("yui-gen1-button")).click();
+    }
+
+    @Test
+    public void testCreateNewView() {
+        createPipelineProject();
+
+        getDriver().findElement((MY_VIEWS)).click();
```
Comment (dmcalifornia) [inline src/test/java/NewViewTest.java:52]: Pls remove the one pair of parenthesis.

File: src/test/java/NewViewTest.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        getDriver().findElement((ADD_TAB)).click();
```
Comment (dmcalifornia) [inline src/test/java/NewViewTest.java:53]: The same as above.

File: src/test/java/NewViewTest.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    private static final String VIEW_NAME = "test_view";
+    private static final By RADIO_BUTTON_MY_VIEW = By.xpath("//*[@id='createItemForm']/div[1]/div[2]/fieldset/div[3]/label");
+    private static final By BUTTON_CREATE = By.id("ok");
+    private static final By PROJECT =
+            By.xpath(String.format("//td/a/span[contains(text(),'%s')]", PIPELINE_NAME));
+
+
+    private void createPipelineProject() {
+        getDriver().findElement(NEW_ITEM).click();
+        getDriver().findElement((PIPELINE)).click();
```
Comment (dmcalifornia) [inline src/test/java/NewViewTest.java:28]: Pls remove one pair of parenthesis.

**Review by dmcalifornia — CHANGES_REQUESTED** [review]

File: src/test/java/NewViewTest.java (hunk 1 above)
Comment (BabintsevaS) [reply src/test/java/NewViewTest.java]: fixed

File: src/test/java/NewViewTest.java (hunk 2 above)
Comment (BabintsevaS) [reply src/test/java/NewViewTest.java]: fixed

File: src/test/java/NewViewTest.java (hunk 3 above)
Comment (BabintsevaS) [reply src/test/java/NewViewTest.java]: fixed

**Review by dmcalifornia — APPROVED** [review]

**Pull request merged**